providers/tests/databricks/hooks/test_databricks.py

```diff
@@ -1309,6 +1309,20 @@ async def test_async_do_api_call_respects_schema(self, mock_get):
         mock_get.assert_called_once()
         assert mock_get.call_args.args == (f"http://{HOST}:7908/api/2.1/foo/bar",)
 
+    @pytest.mark.asyncio
```

Comment:
Do we really need to add another test? Enabling debug logging in any other test will also cover this case, right?

Reply:
Genrally speaking if you add something and you do not add or modify a test for it and the tests succeed before and after your addition - it means that the tests do not test what you added. They might accidentally work but they do not assert that what you added works as expected.

Reply:
Thanks! I will keep in mind to prefer explicitly testing the changes.